src/js/hmrc/README.md

```diff
@@ -13,3 +13,11 @@ Here are some of HMRC's fraud prevention headers required to be sent by web appl
 1. Gov-Client-Local-IPs (A list of all local IPv4 and IPv6 addresses available to the originating device)
 
 Detailed information about each of these headers can be found here:  https://developer.service.hmrc.gov.uk/guides/fraud-prevention/connection-method/web-app-via-server/
+**Details about newly added header Gov-Client-Local-IPs:**
```

Comment:
@Swastyy Thank you for the contribution! The details you added are quite clear, but we want to show just a summary here,  like we did for the other headers above. If someone is interested in further details, they can be found by going to the link shared. 

Can you please check the suggested message in the issue description and add it under other headers at line 14?

Reply:
Is it okay now?

Reply:
I mean the one-line desc with the most important point i.e. input format

Reply:
The header `Gov-Client-Local-IPs-Timestamp` is a new header, not a modification of `Gov-Client-Local-IPs` which already existed. Can you add that description as a new line (line 14)?

Reply:
Done 👍 
Pl review it.